Services/CO.CDP.Organisation.WebApi/Api/Organisation.cs

```diff
@@ -22,6 +22,24 @@ public static class EndpointExtensions
 {
     public static void UseOrganisationEndpoints(this WebApplication app)
     {
+        app.MapGet("/admin/organisations",
```

Comment:
I think we should just be using the `/organisations` endpoint(s) here.

Reply:
I thought it wasn't suitable for this use case since its locked down to a specific user and their access to organisations. Also, this endpoint did not include any pagination properties and thought it high risk to update the existing endpoint.